packages/core/src/lib/createSystem.ts

```diff
@@ -124,6 +124,8 @@ export function createSystem(config: KeystoneConfig, isLiveReload?: boolean) {
           }
         },
         async disconnect() {
+          // Tests that use the stored session won't stop until redis connection is closed
```

Comment:
Redis is just one potential way to store sessions externally, we shouldn't talk about it like this in keystone core

Reply:
Good shout, the comment is now updated